### PR TITLE
kubernetes-cli 1.3.0 (stable), 1.4.0-alpha.0 (devel)

### PR DESCRIPTION
**Pull request by tksm**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

There is the open issue  https://github.com/Homebrew/homebrew-core/pull/1676 but it seems to be neglected and not latest version.

### CODE REVIEW

File: Formula/kubernetes-cli.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def install
 
     dir = "_output/local/bin/darwin/#{arch}"
     bin.install "#{dir}/kubectl"
-    bash_completion.install "contrib/completions/bash/kubectl"
+    (bash_completion/"kubectl").write `#{bin}/kubectl completion bash`
```
Comment (antoineco) [inline Formula/kubernetes-cli.rb]: `kubectl` v1.3.0 does not support the `completion` subcommand as far as I know, so I would avoid doing that before v1.4.0 becomes the new *stable*

Comment (tksm) [inline Formula/kubernetes-cli.rb]: I believe `completion` subcommand was implemenented in v1.3.0-alpha5. I confirmed `completion` works in v1.3.0 (stable) and 1.4.0-alpha.0 (devel).

https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md/#v130-alpha5
> * Move shell completion generation into 'kubectl completion' command ([#23801](https://github.com/kubernetes/kubernetes/pull/23801), [@sttts](https://github.com/sttts))

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/0676761e917fd3ee4354aef98d43b45272b957c7. Thank you for your contribution to Homebrew @tksm; we appreciate it! :smiley_cat:

File: Formula/kubernetes-cli.rb (hunk 1 above)
Comment (antoineco) [inline Formula/kubernetes-cli.rb]: My bad, I missed that commit. Thanks a lot for adding this! 👍

**Pull request closed without merging**